[SamsungTV] NullPointerException during discovery

```
2020-11-03 18:14:41.159 [ERROR] [.core.config.discovery.internal.DiscoveryServiceRegistryImpl] - Cannot notify the DiscoveryListener 'org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler' on Thing discovered event!
java.lang.NullPointerException: null
	at java.util.regex.Matcher.getTextLength(Matcher.java:1770) ~[?:?]
	at java.util.regex.Matcher.reset(Matcher.java:416) ~[?:?]
	at java.util.regex.Matcher.<init>(Matcher.java:253) ~[?:?]
	at java.util.regex.Pattern.matcher(Pattern.java:1133) ~[?:?]
	at org.openhab.binding.samsungtv.internal.WakeOnLanUtility.getMACAddress(WakeOnLanUtility.java:80) ~[?:?]
	at org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler.thingDiscovered(SamsungTvHandler.java:341) ~[?:?]
	at org.openhab.core.config.discovery.internal.DiscoveryServiceRegistryImpl$1.run(DiscoveryServiceRegistryImpl.java:260) ~[bundleFile:?]
	at org.openhab.core.config.discovery.internal.DiscoveryServiceRegistryImpl$1.run(DiscoveryServiceRegistryImpl.java:1) ~[bundleFile:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at org.openhab.core.config.discovery.internal.DiscoveryServiceRegistryImpl.thingDiscovered(DiscoveryServiceRegistryImpl.java:257) [bundleFile:?]
	at org.openhab.core.config.discovery.AbstractDiscoveryService.thingDiscovered(AbstractDiscoveryService.java:266) [bundleFile:?]
	at org.openhab.core.config.discovery.upnp.internal.UpnpDiscoveryService.remoteDeviceAdded(UpnpDiscoveryService.java:160) [bundleFile:?]
	at org.jupnp.registry.RemoteItems$1.run(RemoteItems.java:114) [bundleFile:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
```

--- a/bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/WakeOnLanUtility.java
+++ b/bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/WakeOnLanUtility.java
@@ -77,18 +77,19 @@ public class WakeOnLanUtility {
 
         String[] cmds = Stream.of(COMMAND.split(" ")).map(arg -> String.format(arg, hostName)).toArray(String[]::new);
         String response = ExecUtil.executeCommandLineAndWaitResponse(Duration.ofMillis(CMD_TIMEOUT_MS), cmds);
-        Matcher matcher = MAC_REGEX.matcher(response);
         String macAddress = null;
 
-        while (matcher.find()) {
-            String group = matcher.group();
+        if (response != null) {
+            Matcher matcher = MAC_REGEX.matcher(response);
+            while (matcher.find()) {
+                String group = matcher.group();
 
-            if (group.length() == 17) {
-                macAddress = group;
-                break;
+                if (group.length() == 17) {
+                    macAddress = group;
+                    break;
+                }
             }
         }
-
         if (macAddress != null) {
             LOGGER.debug("MAC address of host {} is {}", hostName, macAddress);
         } else {

